app/models/user.js

```diff
@@ -16,8 +16,9 @@ export default Owner.extend({
   token: attr('string'),
   secureUserHash: attr('string'),
   gravatarId: attr('string'),
-  firstLoggedInAt: attr('date'),
```

Comment:
Same here with regards to the attribute type. The [`date` transform](https://api.emberjs.com/ember-data/release/classes/DateTransform) parses the date string to the `Date` object. Thus, before the change this attribute was a `Date` object, after the change it's just a string, e.g. `2019-11-11T00:50:23Z`